assets/src/blocks/Spreadsheet/SpreadsheetEditor.js

```diff
@@ -2,6 +2,8 @@ import { Component, Fragment } from '@wordpress/element';
 import { InspectorControls } from '@wordpress/block-editor';
 import ColorPaletteControl from '../../components/ColorPaletteControl/ColorPaletteControl';
 
+import { debounce } from 'lodash';
```

Comment:
Definitely good idea to use debounce here, though maybe this PR is not the best place to change the block's behavior.

Reply:
Indeed, it's just that I realized we needed debounce because of the tests!

Reply:
This PR is quite old and approved, no need to create a different one for the `debounce`. :shipit: